src/BirdMessenger/TusBuild.cs

```diff
@@ -64,40 +67,39 @@ public ITusClient Build()
             {
                 configureClientAction(_tusClientOptions);
             }
-            var serviceProvider = _tusClientOptions.Servces.BuildServiceProvider();
+            var serviceProvider = _tusClientOptions.Services.BuildServiceProvider();
 
             var tusClient = serviceProvider.GetService<ITusClient>();
             if (tusClient == null)
             {
                 var tusCore = serviceProvider.GetService<ITusCore>();
                 var tusEx = serviceProvider.GetService<ITusExtension>();
-                tusClient= new TusClient(tusCore,tusEx,
-                    _tusClientOptions.TusHost,_tusClientOptions.GetUploadSize);
+                tusClient = new TusClient(tusCore, tusEx, _tusClientOptions.TusRemoteHost, _tusClientOptions.GetUploadChunkSize);
             }
-            
+
             return tusClient;
         }
-
     }
 
     public class TusClientOption
     {
         /// <summary>
         /// tus server host
         /// </summary>
-        public Uri TusHost { get; set; }
+        public Uri TusRemoteHost { get; set; }
```

Comment:
I think that it is not necessary

Reply:
I agree. It's just a refactor that I can live without.